### PR TITLE
GATEWAY-3273: Added unit tests for core mail package.

### PR DESCRIPTION
**Pull request by mlandis5am**
- unit tests for core mail package.
- reworked poms to inherit versions from dependency-management

### CODE REVIEW

File: Product/Production/Gateway/AdminDistribution_10/pom.xml [hunk 1]
```diff
@@ -49,6 +49,10 @@
```
Comment (msweaver) [inline Product/Production/Gateway/AdminDistribution_10/pom.xml:48]: why is this necessary? Same comment for all of these Gateway project poms.

**Comment by msweaver** [comment]
As a general comment, having stepped back from Direct for a little bit and now coming back to see it again, I am second guessing the placement of this logic into CONNECTCoreLib. Only DirectCore uses this functionality, it seems like all of this would be at home in DirectCore. 

There aren't any prospects of integrating other services into Direct right?

File: Product/Production/Gateway/AdminDistribution_10/pom.xml (hunk 1 above)
Comment (mlandis5am) [inline Product/Production/Gateway/AdminDistribution_10/pom.xml:48]: Either the compile or runtime tests failed without the explicit dependency (I just can't remember which at this moment). I modified the Product/pom.xml below to specify the scope "test" for the spring test jars and after that the jar went missing in these modules without these entries added. AFAIK this is the right way to use maven but I could be missing something here.

**Comment by mlandis5am** [comment]
I updated this to remove the nhincdb.sql changes - they will come separately with 3222.
